toolchain/check/convert.cpp

```diff
@@ -383,12 +384,15 @@ static auto ConvertTupleToTuple(Context& context, SemIR::TupleType src_type,
                                                      new_block.id()});
 }
 
-// Performs a conversion from a struct to a struct type. Does not perform a
-// final conversion to the requested expression category.
-static auto ConvertStructToStruct(Context& context, SemIR::StructType src_type,
-                                  SemIR::StructType dest_type,
-                                  SemIR::NodeId value_id,
-                                  ConversionTarget target) -> SemIR::NodeId {
+// Performs a conversion from a struct to a struct or class type. Does not
+// perform a final conversion to the requested expression category.
+template <typename TargetAccessNodeT>
+static auto ConvertStructToStructOrClass(Context& context,
```

Comment:
Maybe add a hint to make it clearer that this shouldn't be called directly (I assume), or adjust names to more precisely reflect the difference between `ConvertStructToStructOrClass` and `ConvertStructToClass`.

Reply:
Changed comment to say that this is just common implementation for the other two functions.